assembl/static2/js/app/graphql/fragments/Resource.graphql

```diff
@@ -0,0 +1,25 @@
+fragment Resource on Resource {
+  doc {
+    externalUrl
+    mimeType
+    title
+  }
+  embedCode
+  id
+  image {
+    externalUrl
+    mimeType
+    title
+  }
+  text(lang: $lang)
+  title(lang: $lang)
+  titleEntries {
+    localeCode
+    value
```

Comment:
You can use the langStringEntry fragment here.

Reply:
thanks